src/main/java/com/epam/service/UserService.java

```diff
@@ -0,0 +1,64 @@
+package com.epam.service;
+
+import com.epam.dto.UserDTO;
+import com.epam.model.User;
+import com.epam.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import com.epam.mappers.UserMapper;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class UserService {
+
+
+    private final UserRepository userRepository;
+    private final UserMapper userMapper;
+
+    public long getId(UserDTO userDTO) {
+        Optional<User> user = userRepository.findByLogin(userDTO.getLogin());
+        return user.get().getId();
```

Comment:
I'm not really an expert in Spring Data or how it interacts with the Hibernate, but aren't we using Optional here just to throw NoSuchElementException instead of NullPointerException in case we don't have a user with such login?